python/pyspark/ml/tests.py

```diff
@@ -499,6 +499,24 @@ def test_linear_regression(self):
         except OSError:
             pass
 
+    def test_logistic_regression(self):
+        lr = LogisticRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
```

Comment:
Should I make this change to ml/classification.py as well? I have used /lr there as well. 

`lr_path = temp_path + "/lr"`